### ISSUE TITLE
Plone5 - cannot import name ILinkIntegrityInfo

### ISSUE THREAD

**Issue by djowett** [4teamwork/ftw.simplelayout#422]
ZopeXMLConfigurationError: File "/home/daniel/buildout-cache-UCS2/eggs/ftw.simplelayout-1.16.0-py2.7.egg/ftw/simplelayout/configure.zcml", line 21.4-21.34
    ZopeXMLConfigurationError: File "/home/daniel/buildout-cache-UCS2/eggs/ftw.simplelayout-1.16.0-py2.7.egg/ftw/simplelayout/browser/configure.zcml", line 21.4-21.31
    ZopeXMLConfigurationError: File "/home/daniel/buildout-cache-UCS2/eggs/ftw.simplelayout-1.16.0-py2.7.egg/ftw/simplelayout/browser/ajax/configure.zcml", line 20.4-25.10
    ImportError: cannot import name ILinkIntegrityInfo

I'm using Plone 5.0.6, and therefore plone.app.linkintegrity 3.0.6 and that class disappeared in https://github.com/plone/plone.app.linkintegrity/commit/f0a73e8223d615ac6a7bdfab758f94671c6eedb9

**Comment by djowett** [comment]
Maybe [this](https://community.plone.org/t/confirm-deletion-event/2699) helps?

**Comment by maethu** [comment]
@djowett 
We lost trak about the plone 5 compatibility. 
We have only one site running with plone 5 and ftw.simplelayout. I guess there is version 1.1 installed. 

We are currently working on our Plone 5 testing environment to deliver packages with proper plone 5 support. 

Yes, the LinkIntegritiy implementation has been changed from a portal_catalog based solution to a zc.relation solution. Therefore some interfaces has been changed.

**Comment by alyssamuu** [comment]
@maethu 
Hi,

I was wondering if there is any information/plan regarding Plone 5 compatibility? 

We are looking to use and hopefully contribute any extensions on a new Plone 5 project, with the aim for it to be built on ftw.simplelayout.

Thanks!

**Comment by maethu** [comment]
Hey
Glad to hear!
Yes there is a plan now :-)
As I wrote before we had to update our testing stack.

This happened in the last few weeks. Our goal ist to bring Plone 5.1 support until August.
Since we have over 60 Deployments wit ftw.simplelayout we need to do this job really carefully. The main problem will be a proper upgrade path, so this will take some time. 

I'm not yet sure if the release in August will be 5.0.x compatible, since there a few minor issues regarding the plone tests.
